demo/README.md

```diff
@@ -1,6 +1,8 @@
 # How to run a Demo of OSM
 
 ## System Requirements
+- Linux or WSL2 on Windows
+  - running on WSL2 additionally requires gcc
```

Comment:
I would move this out and just add GCC as a requirement. I'm pretty sure I also had to install it.